common/djangoapps/third_party_auth/api/tests/test_views.py

```diff
@@ -1,26 +1,31 @@
+# pylint: disable=no-member
 """
 Tests for the Third Party Auth REST API
 """
-import json
 import unittest
 
 import ddt
-from mock import patch
-from django.test import Client
 from django.core.urlresolvers import reverse
+from mock import patch
+from openedx.core.lib.api.permissions import ApiKeyHeaderPermission
 from rest_framework.test import APITestCase
-from rest_framework import status
 from django.conf import settings
 from django.test.utils import override_settings
-
-from util.testing import UrlResetMixin
-from openedx.core.lib.django_test_client_utils import get_absolute_url
 from social.apps.django_app.default.models import UserSocialAuth
+
 from student.tests.factories import UserFactory
+from third_party_auth.api.permissions import ThirdPartyAuthApiPermission
 from third_party_auth.tests.testutil import ThirdPartyAuthTestMixin
 
 
 VALID_API_KEY = "i am a key"
+IDP_SLUG_TESTSHIB = 'testshib'
+PROVIDER_ID_TESTSHIB = 'saml-' + IDP_SLUG_TESTSHIB
+
+
+def get_mapping_data_by_usernames(usernames):
+    """ Generate mapping data used in response """
+    return [{'username': username, 'remote_id': username} for username in usernames]
```

Comment:
Can you update the test to verify that the `remote_id` is actually returning the remote id and not just the open edX username?  You can do so by updating the call to `UserSocialAuth.objects.create` on L71 and passing in a modified username for the user's remote ID.